src/os/ObjectStore.h

```diff
@@ -411,6 +411,7 @@ class ObjectStore {
       OP_COLL_HINT = 40, // cid, type, bl
 
       OP_TRY_RENAME = 41,   // oldcid, oldoid, newoid
+      OP_SET_COLLOPTS = 42, // cid, pool_opts_t
```

Comment:
I think we can get away with not making this persistent, actually.. and just make sure the OSD calls it on each PG from load_pgs() and when a pool changes.  That is probably less work for the store implementation